blog/prisma-alternative/index.md

```
@@ -0,0 +1,224 @@
+
+## Beyond ORM
+
+A feature-rich ORM can enable some very interesting new use cases. For example, since the ORM is equipped with built-in access control, it can be directly mapped to a service that offers a full-fledged data query API without writing any code. You effectively get a self-hosted Backend-as-a-Service but without any vendor lock-in. Check out the [Query-as-a-Service](/docs/3.x/service) documentation if you're interested.
+
+Further more, [frontend query hooks](/docs/3.x/service/client-sdk/tanstack-query/) (based on [TanStack Query](https://tanstack.com/query)) can be automatically derived and they work seamlessly with the backend service.
+
+All summed up, the project's goal is to be the data layer of modern full-stack applications. Kill boilerplates, eliminate redundancy, and let your data model drive as many things as possible.
+
+## Conclusion
+
+ZenStack V3 is currently in Beta. Please check the documentation [here](https://zenstack.dev/v3) and find the GitHub repo at [https://github.com/zenstackhq/zenstack-v3](https://github.com/zenstackhq/zenstack-v3). Join us in [Discord](https://discord.gg/Ykhr738dUe) to share your thoughts and feedback!
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)